scopes/design/ui/pages/standalone-not-found-page/standalone-not-found.docs.mdx

```diff
@@ -0,0 +1,28 @@
+---
+labels: ['react', 'typescript', '404', 'error', 'page', 'fonts']
+description: 'A 404 error page with fonts'
+---
+
+import { ThemeCompositions } from '@teambit/documenter.theme.theme-compositions';
+import { StandaloneNotFoundPage } from './index';
+
+### Overview
+
+A standalone page component that displays a 404 error message and independently contains the required fonts
+
+```js live
+() => {
+  const accounts = {
+  defAccount: { name: 'defaultAccount', type: 'default', profileImage: 'https://static.bit.dev/harmony/support.svg' },
+  orgAccount: { name: 'defaultAccount', type: 'organization', profileImage: 'https://static.bit.dev/bit-logo.svg' },
+  userAccount: { name: 'defaultAccount', type: 'user', profileImage: 'https://static.bit.dev/harmony/github.svg' },
+};
+
+ return (
+  <ThemeCompositions style={{ minHeight: 90 }}>
+    <StandaloneNotFoundPage />
+  </ThemeCompositions>
+ )
+}
+
+```
```

Comment:
No need for the ThemeCompositions here because the component use the Theme from base-ui, and also load the component icon.
No need for the accounts param also.